include/dirent.h

```diff
@@ -51,7 +53,7 @@ typedef struct dirent {
   uint16_t d_reclen; /* length of this record */
   uint16_t d_namlen; /* length of string in d_name */
   uint8_t d_type;    /* file type, see below */
-  char d_name[0];
+  char d_name[MAXNAMLEN + 1];
```

Comment:
It does not matter, as we use `dirent_t` not to store data, but to exchange it between kernel-space and user-space. However this is a leftover of my other idea, so I'll remove it.